### PR TITLE
Update information for the planned graduation year on MyProfile page

### PR DESCRIPTION
**Pull request by Luke-W-Hart**
Updated the "NOTES" on the MyProfile page for planned graduation date according to Dr. Tuck's suggestions on https://github.com/gordon-cs/gordon-360-api/issues/730 (I couldn't find a way to link it since it was closed and then reopened later).
This is what the changes look like.
Screenshot from 2024-05-28 09-04-53
Screenshot from 2024-05-28 09-05-15

### CODE REVIEW

File: src/components/Profile/components/PersonalInfoList/components/UpdatePlannedGraduationYear/index.jsx [hunk 1]
```diff
@@ -57,9 +57,16 @@ const UpdatePlannedGraduationYear = (props) => {
...
-          NOTE: <br /> This does not set your official graduation <br />
-          year. To make an official change, please <br /> contact{' '}
-          <a href="mailto:registrar@gordon.edu">the Registrar's Office</a>.
+          NOTE: <br /> This does not replace the Application to
+          <br />
+          Graduate.
+          <br />
+          Please fill that out 8-12 months before
+          <br />
+          you plan to graduate. The application <br />
```
Comment (EjPlatzer) [inline src/components/Profile/components/PersonalInfoList/components/UpdatePlannedGraduationYear/index.jsx:66]: The "please" feels like it changes the tone from an important notification to a request. Maybe this is a clearer tone?
```suggestion
          NOTE: <br /> This does not replace the Application to
          <br />
          Graduate, which must be filled out 8-12 months before
          <br />
          you plan to graduate. The application <br />
```

**Review by EjPlatzer — CHANGES_REQUESTED** [review]
This is a nice improvement! I left a couple questions.

File: src/components/Profile/components/PersonalInfoList/components/UpdatePlannedGraduationYear/index.jsx [hunk 2]
```diff
@@ -57,9 +57,16 @@ const UpdatePlannedGraduationYear = (props) => {
...
+          can be found in
+          <a href="https://my.gordon.edu"> my.gordon.edu</a>, in the
+          <br /> Academics tab, on the left.
```
[the remark this answers is not in the document]
Comment (EjPlatzer) [reply src/components/Profile/components/PersonalInfoList/components/UpdatePlannedGraduationYear/index.jsx:69]: I'm guessing the goal is to keep the note from being larger than the main content. If so, that can be accomplished by setting a max-width in CSS, which will allow the content to break more naturally into new lines.

**Comment by Luke-W-Hart** [comment]
Changed the wording and added the max-width.

File: vite.config.ts.timestamp-1716907040051-f567aea7f8b4d.mjs
Comment (EjPlatzer) [inline vite.config.ts.timestamp-1716907040051-f567aea7f8b4d.mjs:1]: This file should not be added.

**Review by EjPlatzer — CHANGES_REQUESTED** [review]

**Pull request merged**